gnc/sub8_trajectory_generator/src/trajectory_generator.cc

```diff
@@ -57,8 +59,30 @@ int main(int argc, char** argv) {
   // Grabs the planner id from the param server
   int planner_id = 0;
   nh.getParam("planner", planner_id);
+
+  ros::ServiceClient sc =
+      nh.serviceClient<sub8_msgs::ThrusterInfo>("thrusters/thruster_range");
+  // Block until this service is available
+  sc.waitForExistence(); 
```

Comment:
Would be wise to roswarn that we're waiting for thruster_range to show up, so future generations see why nothing is happening